Source/BugsnagClient.m

```diff
@@ -1097,7 +1097,7 @@ - (void)updateCrashDetectionSettings {
     if (self.configuration.autoDetectErrors) {
         // Enable all crash detection
         bsg_kscrash_setHandlingCrashTypes(BSG_KSCrashTypeAll);
```

Comment:
I believe this should be checking the enabledErrorTypes as well, rather than enabling all. However, does it even need to update KSCrash now that configuration can't change?

Reply:
You're right - there's no need to update, it looks like this method is obsolete now that configuration is immutable. I've removed this as `BugsnagCrashSentry` performs the calculation (and correctly checks enabledErrorTypes)